daprdocs/content/en/reference/api/jobs_api.md

```diff
@@ -37,6 +37,7 @@ Parameter | Description
 `dueTime` | An optional time at which the job should be active, or the "one shot" time, if other scheduling type fields are not provided. Accepts a "point in time" string in the format of RFC3339, Go duration string (calculated from creation time), or non-repeating ISO8601.
 `repeats` | An optional number of times in which the job should be triggered. If not set, the job runs indefinitely or until expiration.
 `ttl` | An optional time to live or expiration of the job. Accepts a "point in time" string in the format of RFC3339, Go duration string (calculated from job creation time), or non-repeating ISO8601.
+`failure_policy` | An optional failure policy for the job. Details of the format are below.
```

Comment:
Add to this line what the default value is when unset.

Please can we add a new table with the full `failure_policy` API definitions- add what the defaults are when fields are unset.

Reply:
Didn't add a table, but different bullet points for the two fields of the `constant` policy. Do you think this is good enough? I find a table might be confusing given that there are two types of policies, and one of them doesn't have any configuration.